iocore/net/NetProfileSM.cc

```diff
@@ -1,5 +1,7 @@
 /** @file
 
+  A brief file description
+
```

Comment:
sorry, I don't know what should be here, just copy the header from UnixNetVConnection.cc.